roofit/roofitcore/test/CMakeLists.txt

```
@@ -44,3 +44,4 @@ ROOT_ADD_GTEST(testRooProductPdf testRooProductPdf.cxx LIBRARIES RooFitCore)
 ROOT_ADD_GTEST(testNaNPacker testNaNPacker.cxx LIBRARIES RooFitCore)
 ROOT_ADD_GTEST(testRooSimultaneous testRooSimultaneous.cxx LIBRARIES RooFitCore RooFit)
 ROOT_ADD_GTEST(testRooGradMinimizerFcn testRooGradMinimizerFcn.cxx LIBRARIES RooFitCore)
+ROOT_ADD_GTEST(testGlobalObservables testGlobalObservables.cxx LIBRARIES RooFitCore RooFit)
```

```suggestion
ROOT_ADD_GTEST(testGlobalObservables testGlobalObservables.cxx LIBRARIES RooFit)
```
I thought for a long time that you have to mention both, but CMake's transitive dependencies should allow for only mentioning `RooFit`. If it doesn't work, the CMake config needs to be fixed.

---

Oh, but on second look:
This introduces a dependency to RooFit, i.e., the entire set of libRooFit.so files needs to be compiled before you can run this test.

---

True, I tried to avoid this now by using `RooGenericPdf` instead of `RooPoisson` and `RooGaussian`. However, the test gets slowed down by the numeric integration: from 185 ms to about 1.5 seconds. So while I agree that it would be nice to get rid of this dependency, I don't think we should do something about it in this PR. There is also `testRooAbsPdf` and `testRooSimultaneous` that still depend on RooFit too, and I'll try to solve this dependency problem for all of the tests at a later point.